src/app/ui/components/toggle/toggle.tsx

```diff
@@ -0,0 +1,88 @@
+import { type ChangeEvent, useState } from 'react';
+
+import { cva } from 'leather-styles/css';
+import { styled } from 'leather-styles/jsx';
+
+const toggleRecipe = cva({
+  base: {
+    position: 'relative',
+    display: 'inline-block',
+    width: '42px',
+    height: '24px',
+    '& input': {
+      opacity: 0,
+      width: 0,
+      height: 0,
+    },
+    '& .slider': {
+      position: 'absolute',
+      cursor: 'pointer',
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: 0,
+      backgroundColor: 'ink.text-subdued',
+      border: '1px solid #F5F1ED',
```

Comment:
Can you use our design tokens for colours, spacing, border etc. ? 

https://github.com/leather-io/mono/tree/dev/packages/tokens/src

Reply:
added for some where possible, but some specific pixel sizes dont exist such as 42px (width)
@pete-watters